src/Entity/EntityDocument.php

```diff
@@ -2,11 +2,13 @@
 
 namespace Wikibase\DataModel\Entity;
 
+use InvalidArgumentException;
+
 /**
  * Minimal interface for all objects that represent an entity.
  * All entities have an EntityId and an entity type.
  *
- * @since 0.8.2, modified in 3.0
+ * @since 0.8.2
```

Comment:
I think this does not belong here. This line should say since when a class exists. If a method signature changed, the since tag of this method should change, not the since tag of the class.

Feel free to click edit and revert my change in this line. It's unrelated to the rest of this patch anyway.